README.md

```diff
@@ -93,7 +94,7 @@ Example using a schema registry:
 Avromatic.configure do |config|
   config.schema_store = AvroTurf::SchemaStore.new(path: 'avro/schemas')
   config.registry_url = Rails.configuration.x.avro_schema_registry_url
-  config.build_schema_registry!
+
```

Comment:
Can we add a comment that `build_messaging!` will automatically call `build_schema_registry!` if a `schema_registry` hasn't been configured?

Reply:
done, I added an explicit note right under the example, is that what you were thinking?